rules/internal/framework_middleman.bzl

```diff
@@ -121,7 +163,9 @@ def _framework_middleman(ctx):
 
 framework_middleman = rule(
     implementation = _framework_middleman,
-    attrs = dicts.add(split_transition_rule_attrs, {
+    fragments = ["cpp"],
+    toolchains = ["@bazel_tools//tools/cpp:toolchain_type"],
```

Comment:
Can you update this to use `use_cpp_toolchain` instead, similarly to what i did in #845